Comments/ClientApp/src/components/PhaseBanner/PhaseBanner.js

```diff
@@ -18,13 +18,19 @@ const getBrowserInfo = () => {
 	return environmentName;
 };
 
+const phaseBannerClicked = () => {
+	if(window.location.hostname.includes("alpha")) {
```

Comment:
just an FYI - includes doesn't work in older browsers but we're polyfilling it from core-js (this line "import "core-js/fn/array/includes";" in src/index.js